docs/blog/announcing-vite6.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Vite 6.0 is out!
+author:
+  name: The Vite Team
+date: 2024-11-26
+sidebar: false
+head:
+  - - meta
+    - property: og:type
+      content: website
+  - - meta
+    - property: og:title
+      content: Announcing Vite 6
+  - - meta
+    - property: og:image
+      content: https://vite.dev/og-image-announcing-vite6.png
+  - - meta
+    - property: og:url
+      content: https://vite.dev/blog/announcing-vite6
+  - - meta
+    - property: og:description
+      content: Vite 6 Release Announcement
+  - - meta
+    - name: twitter:card
+      content: summary_large_image
+---
+
+# Vite 6.0 is out!
+
+_November 26, 2024_
+
+![Vite 6 Announcement Cover Image](/og-image-announcing-vite6.png)
+
+It has been an eventful year. Vite adoption keeps growing, with npm downloads per week jumping from 7.5 million to 16 million since the release of Vite 5 a year ago. [Vitest](https://vitest.dev) is not only being chosen by final users but is also starting to form an ecosystem of its own. For example, [Storybook](https://storybook.js.org) has new testing capabilities powered by Vitest. New frameworks have joined the Vite ecosystem. Among others: [TanStack Start](https://tanstack.com/start), [One](https://onestack.dev/), [Ember](https://emberjs.com/). Web frameworks are innovating at an increasingly faster pace. You can check out the improvements folks have been doing at [Astro](https://astro.build/), [Nuxt](https://nuxt.com/), [SvelteKit](https://kit.svelte.dev/), [Solid Start](https://www.solidjs.com/blog/introducing-solidstart), [Qwik City](https://qwik.builder.io/qwikcity/overview/), [RedwoodJS](https://redwoodjs.com/), [Remix](https://remix.run/), and the list goes on.
+
```

Comment:
With the big [React Router v7](https://remix.run/blog/react-router-v7) release yesterday, I imagine it's helpful to the remix team if it's React Router that's listed here rather than Remix. It would align with the other entries naming the server stack rather than the teams behind. Remix is still mentioned below in the Bolt section too.

Reply:
Good point, let's change it. Thanks!